synapse/storage/schema/main/delta/76/06_rooms_to_purge.sql

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2023 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- cf upsert_room_to_purge docstring for the meaning of the fields.
+CREATE TABLE IF NOT EXISTS rooms_to_purge(
+    room_id text NOT NULL,
+    delete_id text NOT NULL,
+    "status" text NOT NULL,
+    "error" text,
+    "timestamp" bigint,
+    "shutdown_params" text,
+    "shutdown_response" text,
```

Comment:
Quoted vs unquoted column names :shrug: 

Looks like we prefer unquoted almost everywhere else

Reply:
So some of them are keywords hence the need for quotes, but then I did it for the ones I added after too. I'll check and unquote what can be unquote.

Reply:
Well, VSCode is highlighting them like keywords, but it works so it seems not true. Updated with no quotes.